### PR TITLE
ICDS Dashboard

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @emord,

Changes for cases: 
http://manage.dimagi.com/default.asp?271141 - I fix awc location filter to send parameters in correct format.
http://manage.dimagi.com/default.asp?271011 - I change strings in all places
http://manage.dimagi.com/default.asp?271156 - I reverse arrow colors for this indicator.

Please let me know if you have any questions.

Regards,
Łukasz

### CODE REVIEW

**Comment by lwyszomi** [comment]
@emord  I added changes for:
http://manage.dimagi.com/default.asp?271239
http://manage.dimagi.com/default.asp?271240
http://manage.dimagi.com/default.asp?271243
http://manage.dimagi.com/default.asp?271244

These are a small UI changes for PDF report.

Please let me know if you have any questions.

Regards,
Łukasz

File: custom/icds_reports/sqldata.py [hunk 1]
```diff
@@ -2232,7 +2232,8 @@ def new_table_config(self):
                                 'header': 'Percent of children born in month with low birth weight',
                                 'slug': 'low_birth_weight',
                                 'average': [],
-                                'format': 'percent'
+                                'format': 'percent',
+                                'reverseColors': True,
```
Comment (emord) [inline custom/icds_reports/sqldata.py]: what does reverseColors do?

File: custom/icds_reports/templates/icds_reports/icds_app/download.directive.html [hunk 2]
```diff
@@ -243,8 +243,8 @@
             </div>
             <div class="issnip-spinner-container" ng-if="($ctrl.showProgressBar() || $ctrl.readyToDownload() || $ctrl.queuedISSNIPTask) && !$ctrl.downloaded">
                 <i ng-if="$ctrl.showProgressBar() || $ctrl.queuedISSNIPTask" class="fa fa-circle-o-notch fa-spin"></i>
-                <span ng-if="$ctrl.showProgressBar() || $ctrl.queuedISSNIPTask">Preparing your ISSNIP report...</span>
-                <span ng-if="$ctrl.readyToDownload()">Your ISSNIP report is ready to be downloaded</span>
+                <span ng-if="$ctrl.showProgressBar() || $ctrl.queuedISSNIPTask">Preparing your ICDS-CAS montlhy register report...</span>
+                <span ng-if="$ctrl.readyToDownload()">Your ICDS-CAS montlhy register report is ready to be downloaded</span>
```
Comment (emord) [inline custom/icds_reports/templates/icds_reports/icds_app/download.directive.html]: monthly is misspelled

File: custom/icds_reports/static/js/directives/download/download.directive.js [hunk 3]
```diff
@@ -311,6 +311,8 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
     };
 
     vm.submitISSNIPForm = function(csrf_token) {
+        $rootScope.issnip_report_link = '';
```
Comment (emord) [inline custom/icds_reports/static/js/directives/download/download.directive.js]: Why does this need to be reset?

**Review by emord — COMMENTED** [review]
Went ahead and fixed the spelling on your branch. Other comments are non blocking

File: custom/icds_reports/static/js/directives/download/download.directive.js (hunk 3 above)
Comment (lwyszomi) [reply custom/icds_reports/static/js/directives/download/download.directive.js]: @emord When I tested PDF report I noticed that when you try generate next report then we have two messages.

File: custom/icds_reports/sqldata.py (hunk 1 above)
Comment (lwyszomi) [reply custom/icds_reports/sqldata.py]: @emord 
This change arrow colors, if ```reverseColors``` is ```True``` then up arrow will be red and down arrow will be green

**Comment by lwyszomi** [comment]
@emord thanks that you fixed typo in monthly

**Pull request merged**